config.js

```diff
@@ -1,13 +1,11 @@
 const dev = process.env.NODE_ENV === `development`
-const stargate = process.env.STARGATE || `http://localhost:9071`
-const graphql = process.env.VUE_APP_GRAPHQL_URL || `localhost:4000`
+const stargate = process.env.STARGATE || `http://localhost:9070`
 
 export default {
   name: `Lunie`,
   development: dev,
-  network: `cosmos-hub-mainnet`,
+  network: process.env.NETWORK || `cosmos-hub-mainnet`,
   stargate,
-  graphql,
```

Comment:
can you explain your thinking here?

Reply:
this prop is not used and replicated a couple of lines more down